src/extension/module.js

```diff
@@ -2429,37 +2446,41 @@
    * @private
    * @param {Sidekick} sk The sidekick
    */
-  async function showSpecialView(sk) {
+  async function showView(sk) {
+    if (!sk.isProject()) {
+      return;
+    }
     const {
-      config: {
-        lang,
-        specialView,
-      },
       location: {
         origin,
         href,
       },
     } = sk;
-    if (specialView && !getSpecialViewOverlay(sk)) {
-      // hide original content
-      [...sk.parentElement.children].forEach((el) => {
-        if (el !== sk) {
-          try {
-            el.style.display = 'none';
-          } catch (e) {
-            // ignore
-          }
-        }
-      });
-      const { viewer, title, titleI18n } = specialView;
+    const searchParams = new URLSearchParams(window.location.search);
+    if (searchParams.get('sidekickResource')) {
+      // custom view
+      return;
+    }
+    const [view] = findViews(sk, VIEWS.DEFAULT);
+    if (view && !getViewOverlay(sk)) {
+      const { viewer, title } = view;
       if (viewer) {
         const viewUrl = new URL(viewer, origin);
         viewUrl.searchParams.set('url', href);
-        const viewOverlay = getSpecialViewOverlay(sk, true);
-        viewOverlay.querySelector('.title').textContent = (titleI18n && titleI18n[lang])
-          || title || i18n(sk, 'json_view_description');
+        const viewOverlay = getViewOverlay(sk, true);
+        viewOverlay.querySelector('.title').textContent = title(sk);
         viewOverlay.querySelector('.container')
           .setAttribute('src', viewUrl.toString());
+        // hide original content
+        [...sk.parentElement.children].forEach((el) => {
+          if (el !== sk) {
+            try {
+              el.style.display = 'none';
+            } catch (e) {
+              // ignore
+            }
```

Comment:
Why do you need to hide the original content ? What do you need to hide... ? Why showing "it" in a first place ?
If really necessary, maybe using a css class would be nicer...

Reply:
This part isn't new, just got moved slightly within the same function. I hide the original to avoid scrolling issues in the overlay. We can simplify this, but I wanted to change as little as possible in this PR.